coderedcms/blocks/html_blocks.py

```diff
@@ -84,6 +90,7 @@ class EmbedGoogleMapBlock(BaseBlock):
         )
     )
 
+
```

Comment:
The build caught a minor style guide violation here...

You can run it locally to test and ensure it passes by doing: `./ci/run_flake8.ps1`

![image](https://user-images.githubusercontent.com/13453401/62251190-f3fa4700-b3bc-11e9-9df2-c06d77438a40.png)